databaseconnect.py

```diff
@@ -68,39 +84,43 @@ def add_to_database(classification, subject, root, verb, H):
     db = connection_to_database()
     cur = db.cursor()
     cur = db.cursor(buffered=True)
-    if classification == "C":
-        cur.execute(
-            f"INSERT INTO chat_table(root_word,verb,sentence) VALUES ('{root}','{verb}','{H}')"
-        )
-        db.commit()
-    elif classification == "Q":
-        cur.execute("SELECT sentence FROM question_table")
-        res = cur.fetchall()
-        exist = 0
-        for r in res:
-            if r[-1] == H:
-                exist = 1
-                break
-        if exist == 0:
-            # do not add if question already exists
+    try:
+        if classification == "C":
             cur.execute(
-                f"INSERT INTO question_table(subject,root_word,verb,sentence) VALUES ('{subject}','{root}','{verb}','{H}')"
+                f"INSERT INTO chat_table(root_word,verb,sentence) VALUES ('{root}','{verb}','{H}')"
             )
             db.commit()
+        elif classification == "Q":
+            cur.execute("SELECT sentence FROM question_table")
+            res = cur.fetchall()
+            exist = 0
+            for r in res:
+                if r[-1] == H:
+                    exist = 1
+                    break
+            if exist == 0:
+                # do not add if question already exists
+                cur.execute(
+                    f"INSERT INTO question_table(subject,root_word,verb,sentence) VALUES ('{subject}','{root}','{verb}','{H}')"
+                )
+                db.commit()
+        else:
+            cur.execute("SELECT sentence FROM statement_table")
+            res = cur.fetchall()
+            exist = 0
+            for r in res:
+                if r[-1] == H:
+                    exist = 1
+                    break
+            if exist == 0:  # do not add if question already exists
+                cur.execute(
+                    f"INSERT INTO statement_table(subject,root_word,verb,sentence) VALUES ('{subject}','{root}','{verb}','{H}')"  # noqa: E501
+                )
+                db.commit()
+    except Exception as e:
+        return e
     else:
-        cur.execute("SELECT sentence FROM statement_table")
-        res = cur.fetchall()
-        exist = 0
-        for r in res:
-            if r[-1] == H:
-                exist = 1
-                break
-        if exist == 0:  # do not add if question already exists
-            cur.execute(
-                f"INSERT INTO statement_table(subject,root_word,verb,sentence) VALUES ('{subject}','{root}','{verb}','{H}')"
-            )
-            db.commit()
-    return db
+        return "Success"
```

Comment:
Introduced a `try-except-else` block to return any `Exception`s from `cur.execute` statements, "`Success`" otherwise